app/src/main/java/com/endava/myendava/repositories/SignInRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.endava.myendava.repositories;
+
+import com.endava.myendava.models.LoginResult;
+import com.endava.myendava.rest.RetrofitClient;
+
+import io.reactivex.Observable;
+
+public class SignInRepository {
+    private final RetrofitClient mRetrofitClient;
+
+    public SignInRepository(RetrofitClient mRetrofitClient) {
+        this.mRetrofitClient = mRetrofitClient;
+    }
+
+    public Observable<LoginResult> isUserExistent(String email) {
```

Comment:
this is not renamed to login ... but if you do, please also rename the file to LoginRepository (my preference would be login and not signIn, since in the bigger picture of the system's architecture, we refer to this action as login and might cause confusion if called signIn)